distribution/docs/src/main/resources/_contents/_sources/solr-embedded-catalog-provider-contents.adoc

```diff
@@ -0,0 +1,38 @@
+
+==== Solr Embedded Catalog Provider
+
+The Solr Embedded Provider is an embedded, local Solr Server instance used in conjunction with an Apache Solr Server data store.
```

Comment:
This is the last one I'll point out until we decide if it should be all references should omit the term Catalog :-) Another good example of differing terms though, where this talks about the Solr Embedded Provider while the title above it is Solr Embedded Catalog Provider